lnbits/core/views/api.py

```diff
@@ -172,7 +172,7 @@ async def api_create_wallet(
 
 @api_router.post("/api/v1/account", response_model=Wallet)
 async def api_create_account(data: CreateWallet) -> Wallet:
-    if len(settings.lnbits_allowed_users) > 0:
+    if not settings.lnbits_create_account or len(settings.lnbits_allowed_users) > 0:
```

Comment:
- nitpick: add property to settings:
```py
@property
def allow_new_user_creation(self):
   return settings.lnbits_create_account and len(settings.lnbits_allowed_users) == 0
```

Reply:
addressed in b9e6b915c477abdf5d3267ac4a5fa3b221c0ffaf